test/unit/classes/instanceof.spec.ts

```diff
@@ -49,6 +49,28 @@ test("null instanceof Class", () => {
     `.expectToMatchJsResult();
 });
 
+test("function instance of Class", () => {
```

Comment:
```suggestion
test("function instanceof Class", () => {
```

Same applies to other new tests